src/EditorFeatures/Core.Wpf/Completion/Presentation/CompletionPresenterSession.cs

```diff
@@ -20,6 +20,8 @@ namespace Microsoft.CodeAnalysis.Editor.Implementation.IntelliSense.Completion.P
 
     internal sealed class CompletionPresenterSession : ForegroundThreadAffinitizedObject, ICompletionPresenterSession
     {
+        public static readonly object DocumentKey = new object();
```

Comment:
Just add a comment what this is used for (i.e. it's a key added to which type?)

Reply:
done!